rai/node/rpc.cpp

```diff
@@ -1790,6 +1790,11 @@ class history_visitor : public rai::block_visitor
 					tree.put ("subtype", "change");
 				}
 			}
+			else if (raw && balance == previous_balance && !handler.node.ledger.epoch_link.is_zero () && block_a.hashables.link == handler.node.ledger.epoch_link)
```

Comment:
The `raw` condition needs to be in a separate if statement, like with the change condition above, otherwise it'll fall through to the receive code.